### DISCUSSION TITLE
Napalm status tab

### DISCUSSION THREAD

**Discussion by bogdancordos** [netbox-community/netbox#7225] [Q&A]
Hi,

Since I upgraded my NetBox from 3.0.0 to 3.0.1, the "Status" tab on the device view doesn't get populated. There is no error returned, and the "Configuration" tab successfully pulls the config from the device, so napalm seems to be working fine.
Has anyone come across this, or can you point me where should I look for detailed logs?

Thanks

**Comment by candlerb** [comment]
It works for me on 3.0.2 - checked with a Mikrotik router using napalm-ros.

![image](https://user-images.githubusercontent.com/44789/132685472-4ea9b4a4-a666-4d36-a522-e9112b9f6594.png)

LLDP Neighbors works too, although Configuration doesn't: `Method get_config not implemented for NAPALM driver <class 'napalm_ros.ros.ROSDriver'>`

(Aside: looks like the °F value should be [rounded to one decimal place](https://github.com/netbox-community/netbox/issues/7228)!)

So I suggest you first check 3.0.2, although I don't see anything in the release notes. Also do a shift-reload in your browser to refresh Javscript.  If it still doesn't work, you could provide more details about your device type / OS and napalm driver.  Open your browser's developer console, go to the device status tab in Netbox, and see if any errors are shown in the Network or Console sections.  If that doesn't show anything, then manually invoke the napalm calls that Netbox makes and see if they succeed.

**Reply by bogdancordos** [reply]
Thank you,

I did upgrade to 3.0.2, however, the issue is still present, and I get the fallowing error:

On Firefox: 
Neither "Facts" or "Environment" gets populated.
![image](https://user-images.githubusercontent.com/57483807/132713667-f1bf6930-6f55-4dc7-8f00-1b2232876124.png)

On Edge:
"Facts" gets populated, "Environment" don't.
Error: "Method get_environment failed: argument of type 'int' is not iterable"

I need to mention that both "LLDP Neighbors" and "Configuration" works fine.
I'm only using "ios" Napalm driver for ISR4321 and WS-C3850

Will "debug" on give me more details on what is going on?

Thanks

**Reply by candlerb** [reply]
DEBUG in configuration.py? I don't think so.  Looks like some brokenness in the Javascript which makes Ajax requests to the Netbox API, but I'm afraid I can't help you more than that.  If someone can replicate this in their own environment and submit a PR, I think it would be likely to be accepted.

Have you tried Chrome?  That's what I'm using, and I find its debug console is helpful.
